manticoresearch_text_embeddings.h

```diff
@@ -9,68 +9,48 @@
 #include <ostream>
 #include <new>
 
-using TextModelWrapper = void*;
+struct TextModelResult {
+  void *model;
+  char *error;
```

Comment:
m.b. `const char *m_szError` ? 
We don't expect c++ side will modify pointee. And 'sz' common for asciiz strings